eng/build.ps1

```diff
@@ -752,7 +752,7 @@ try {
   if ($bootstrap -and $bootstrapDir -eq "") {
     Write-Host "Building bootstrap Compiler"
     Exec-Script (Join-Path $PSScriptRoot "make-bootstrap.ps1") "-name build -force -ci:$ci"
-    $bootstrapDir = Join-Path $ArtifactsDir "bootstrap" "build"
+    $bootstrapDir = Join-Path (Join-Path $ArtifactsDir "bootstrap") "build"
```

Comment:
Why are we doing this? Our scripts are `pwsh` compatible not powershell.

Reply:
@jaredpar Our internal queue for doing test VS insertions broke, so I did this as a workaround. This

> Our scripts are `pwsh` compatible not powershell. 

implies you are knowledgeable in the ways of PowerShell and can thus tell me how to have fixed the queue instead. :smile: